examples/ui-demo/src/components/shared/ExternalLink.tsx

```diff
@@ -1,20 +1,22 @@
 import Link, { LinkProps } from "next/link";
+import { forwardRef, ReactNode } from "react";
 
-const ExternalLink = ({
-  className,
-  children,
-  ...rest
-}: LinkProps & { className?: string; children: React.ReactNode }) => {
+const ExternalLink = forwardRef<
```

Comment:
another noob question: what does `forwardRef` do for us here? Is this something specific to nextjs / ssr?

Reply:
radix tooltip needs to be able to forward the ref to whatever we have as the tooltip trigger, so that they can use it for whatever they want to do w/ it.